CODE_OF_CONDUCT.md

```diff
@@ -60,7 +61,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-sahoosagnik1@gmail.com.
+[INSERT CONTACT METHOD].
```

Comment:
Add the contact method 😑 i.e., sahoosagnik1@gmaail.com

Reply:
Done